gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/Constants.java

```diff
@@ -120,6 +120,22 @@
 
     public static final String LEVEL_CUSTOM = "custom";
 
+    public static final String RETURN_VALUE = "returnVariableKey";
+
```

Comment:
Please document these contants, believe me that will come handy in 2-5 years down the line when we are supporting the product.